### ISSUE TITLE
[BUG] - Custom layer is not showing in custom map

### ISSUE THREAD

**Issue by apanizza** [starschema/mapboxgl-powerbi#82]
Hi, 

I can't manage to get a custom layer to be displayed when accessed via PowerBI. The data is displayed properly on the map and the map is displayed properly (except the custom layer). The custom layer is visible in the MapBox creation maps tools.

There is one error I can see in the dev console which may be the cause of my problem ?

![image](https://github.com/starschema/mapboxgl-powerbi/assets/13343846/0a1ebd3d-4d25-40ba-a525-43fec501ec21)

**Comment by drakvuf** [comment]
The plugin currently using mapbox-gl **2.10** and the [pitch expression](https://docs.mapbox.com/style-spec/reference/expressions/#pitch) you are trying to use is available from version 2.16.
So, we currently not support this feature.

**Comment by apanizza** [comment]
Thanks for your reply. Is there any workaround to get the mapbox layer to work in PowerBI ? I used it few years ago without any issue. Not sure what to change to get it to work again.

**Comment by drakvuf** [comment]
You are right, it should be working with this version, I misread the documentation.
Can you share your custom map style to help us resolve this issue?

**Comment by apanizza** [comment]
Thanks for your help.

You can access the custom map in Mapbox here:
[mapbox://styles/apanizza/clny208fq003k01rfa1px92if](url)

The tileset ID:
apanizza.awyzwjef

I use a shapefile (Maritime Boundaries) from Marine Regions (https://www.marineregions.org/downloads.php) 

Thanks again
